posthog/clickhouse/client/connection.py

```diff
@@ -156,8 +157,7 @@ def default_client(host=settings.CLICKHOUSE_HOST):
     )
 
 
-@cache
-def make_ch_pool(**overrides) -> ChPool:
+def _make_ch_pool(*, client_settings: Mapping[str, str] | None = None, **overrides) -> ChPool:
```

Comment:
This rename was necessary to allow bypassing the pool cache: the client settings map isn't hashable and therefore not usable in a cache key.